src/NearOrg/Notifications/Notification.jsx

```diff
@@ -130,6 +133,7 @@ const path = item.path || "";
 
 // Build notification
 let { blockHeight, accountId } = props;
+let { manageNotification } = props;
```

Comment:
```suggestion
let { blockHeight, accountId, manageNotification } = props;
```

Reply:
This has been addressed in the latest commit. Thank you